google-cloud-storage/src/main/java/com/google/cloud/storage/GrpcStorageImpl.java

```diff
@@ -698,56 +700,66 @@ public Blob compose(ComposeRequest composeRequest) {
 
   @Override
   public CopyWriter copy(CopyRequest copyRequest) {
```

Comment:
this is a confusing diff but basically the try,catch and finally block are new and the rest has been indented within the try block unchanged.